src/VisualStudio/Xaml/Impl/project.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "dependencies": {
+    "NuGet.VisualStudio": {
```

Comment:
Good point. I revisited this and got away with just the following two NuGet dependencies:
    "Microsoft.VisualStudio.Editor": "14.3.25407",
    "Microsoft.VisualStudio.Shell.14.0": "14.3.25407"